nexus/db-queries/src/db/datastore/inventory.rs

```diff
@@ -2264,6 +2301,40 @@ impl DataStore {
             map.zones.zones.push(zone);
         }
 
+        // Now load the clickhouse keeper cluster memberships
+        let clickhouse_keeper_cluster_membership = {
+            use db::schema::inv_clickhouse_keeper_membership::dsl;
+            let mut memberships = BTreeSet::new();
+            let mut paginator = Paginator::new(batch_size);
+            while let Some(p) = paginator.next() {
+                let batch = paginated(
+                    dsl::inv_clickhouse_keeper_membership,
+                    dsl::queried_keeper_id,
+                    &p.current_pagparams(),
+                )
+                .filter(dsl::inv_collection_id.eq(db_id))
+                .select(InvClickhouseKeeperMembership::as_select())
+                .load_async(&*conn)
+                .await
+                .map_err(|e| {
+                    public_error_from_diesel(e, ErrorHandler::Server)
+                })?;
+                paginator = p.found_batch(&batch, &|row| row.queried_keeper_id);
+                for membership in batch.into_iter() {
+                    memberships.insert(
+                        ClickhouseKeeperClusterMembership::try_from(membership)
+                            .map_err(|e| {
+                                Error::internal_error(&format!(
+                                    "{:#}",
+                                    e.to_string()
```

Comment:
I think you want
```suggestion
                                    "{e:#}",
```
here; `"{:#}", e.to_string()` is the equivalent of `"{}", e`.

Reply:
Fixed! Thanks!